saleor/plugins/webhook/utils.py

```diff
@@ -38,6 +47,18 @@ def from_payment_app_id(app_gateway_id: str) -> Optional["PaymentAppData"]:
     return None
 
 
+def from_shipping_app_id(app_shipping_method_id: str) -> Optional["ShippingAppData"]:
+    splitted_id = app_shipping_method_id.split(":")
+    if len(splitted_id) == 3 and splitted_id[0] == APP_ID_PREFIX and all(splitted_id):
```

Comment:
Instead of repeating this if statement we could create function for that. In any case it will required only one place to change - easier to maintain in the future :)